### DISCUSSION TITLE
Export Recipe GIF?

### DISCUSSION THREAD

**Discussion by SamsTheNerd** [shedaniel/RoughlyEnoughItems#1178] [Q & A] [answered]
howdy, I was wondering if there's a way to export a gif of a recipe for recipes that can accept multiple items, like ones with glass for example.

I looked around in the menus and couldn't find anything so I assume it's not a thing.

It could be a neat feature though ? since it better communicates that there's different options, and would be good for mod documentation and whatnot.

Although just taking a couple of still exports and combining them into a gif manually isn't that hard anyways

**Comment by shedaniel** [comment, marked as answer]
Thanks, I am now tracking this suggestion at #1280
